code/modules/mining/lavaland/necropolis_chests.dm

```diff
@@ -404,11 +397,11 @@
 /obj/vehicle/lavaboat/dragon
 	name = "mysterious boat"
 	desc = "This boat moves where you will it, without the need for an oar."
-	keytype = null
 	icon_state = "dragon_boat"
-	generic_pixel_y = 2
-	generic_pixel_x = 1
-	vehicle_move_delay = 1
+
+/obj/vehicle/lavaboat/dragon/buckle_mob()
+	..()
```

Comment:
whereever you can try to make it . = ..()

this helps future developers because if they ever want to change buckle_mob to return a value they can do it easier.